package.json

```diff
@@ -62,7 +69,7 @@
     "webpack-dev-server": "^3.11.0"
   },
   "peerDependencies": {
-    "p5": "*",
+    "p5": "^1.1.9",
```

Comment:
Do we need a specific version definition?

Reply:
I would like to avoid updating the main dependency in the current PR and prefer to do it in a separate one.